python/tests/test_wright_fisher.py

```diff
@@ -38,12 +38,14 @@
 
 class WrightFisherSimulator:
     """
```

Comment:
I guess I have to test the new behavior (multiple pops and migration). any ideas what kinds of tests are needed? @petrelharp

Reply:
Hm. Check where else this is tested? But - we're using this to produce wierd tree sequences for testing, not as a validated Wright-Fisher simulator, so we don't have to check for statistical correctness, just validity of the tree sequence, really. So: does it run? does the tree sequence have more than one population? does it have samples in different populations? That sort of thing.